### PR TITLE
Activity 3 – Online Resume of Kareen Collamat - Advance 

### PR DESCRIPTION
**Pull request by kareencoll**

### CODE REVIEW

File: collamat_kareen/Activity3/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+					<h3>Front End Developer</h3>
+					Ocean Swell Digital<br />					
+					2021 - Present
+					<h3>Freelance Front End Developer</h3>
+					PBN Sites, Local Pasty Shop, On Call Maintenance<br />					
+					2018 - 2021
+				</div>
+			</div>
+			<div style="clear:both;"></div>
+			<div class="content2">
```
Comment (mrmartincarlito) [inline collamat_kareen/Activity3/index.html:46]: Kindly rename this container content on more descriptive way

class="training-attended"
class="lower-section"

naming must be descriptive

File: collamat_kareen/Activity3/index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+				<div class="three">
```
Comment (mrmartincarlito) [inline collamat_kareen/Activity3/index.html:47]: same with this, do not use variable class name with numbers

File: collamat_kareen/Activity3/style.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+body{
+	color: #473C38;
+	font-family: Open Sans;
+}
+a{
```
Comment (mrmartincarlito) [inline collamat_kareen/Activity3/style.css:5]: avoid global tag css declaration
please use class instead

File: collamat_kareen/Activity3/style.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	border-bottom: 2px solid #AE7E5E;
+	width: 100%;
+	padding-top: 80px;
+}
+.upperfold-content{
+	height: 100%;
+	margin: auto;
+	width: 1100px;
+}
+img{
```
Comment (mrmartincarlito) [inline collamat_kareen/Activity3/style.css:34]: Please do not use global tag css declaration please use class instead

File: collamat_kareen/Activity3/index.html (hunk 1 above)
Comment (kareencoll) [reply collamat_kareen/Activity3/index.html]: This is done, sir.

File: collamat_kareen/Activity3/index.html (hunk 2 above)
Comment (kareencoll) [reply collamat_kareen/Activity3/index.html]: Updated, sir.

File: collamat_kareen/Activity3/style.css (hunk 3 above)
Comment (kareencoll) [reply collamat_kareen/Activity3/style.css]: Noted, sir. Thank you. Updated.

File: collamat_kareen/Activity3/style.css (hunk 4 above)
Comment (kareencoll) [reply collamat_kareen/Activity3/style.css]: Updated, sir.

**Pull request merged**